Initialtask/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.3"
+services:
+  mysql:
+    image: mysql:5.7
+    hostname: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: importTest
+      MYSQL_USER: dev
+      MYSQL_PASSWORD: dev
+      VIRTUAL_HOST: mysql.dev.loc
+    ports:
+      - "3306:3306"
+    volumes:
+      - ./docker:/docker-entrypoint-initdb.d
+  php-fpm:
+    image: webdevops/php-dev:7.4
```

Comment:
use official php image https://hub.docker.com/_/php
If you don't need UI part use cli image only

Reply:
When I trying use oficial Image. Container is stopping after 40-50 seconds uptime. 
	I am compelled use working image(not oficial).